flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/DefaultMultipleComponentLeaderElectionServiceTest.java

```diff
@@ -203,6 +206,68 @@ public void allKnownLeaderInformationCallsEventHandlers() throws Exception {
         }
     }
 
+    @Test
+    public void allKnownLeaderInformationDoesNotBlock() throws Exception {
+        final TestingMultipleComponentLeaderElectionDriver leaderElectionDriver =
+                TestingMultipleComponentLeaderElectionDriver.newBuilder().build();
+        final DefaultMultipleComponentLeaderElectionService leaderElectionService =
+                new DefaultMultipleComponentLeaderElectionService(
+                        fatalErrorHandlerExtension.getTestingFatalErrorHandler(),
+                        new TestingMultipleComponentLeaderElectionDriverFactory(
+                                leaderElectionDriver),
+                        java.util.concurrent.Executors.newSingleThreadScheduledExecutor());
```

Comment:
static import?

Reply:
I think it is helpful to see where static methods come from. Therefore, I would not do this change.